test/unit/uk/gov/hmrc/bindingtariffclassification/model/search/SearchTest.scala

```diff
@@ -53,7 +54,14 @@ class SearchTest extends UnitSpec {
     }
 
     "Unbind Populated Search to Query String" in {
-      val populatedQueryParam: String = "queue_id=queue-id&assignee_id=assignee-id&status=status&trader_name=trader-name&sort_by=days-elapsed&sort_direction=desc"
+      val populatedQueryParam: String =
+        "queue_id=queue-id" +
+          "&assignee_id=assignee-id" +
+          "&status=NEW" +
+          "&status=OPEN" +
```

Comment:
Looks like as status is NEW **and** OPEN , are we not sending them like a list?  
`"&status=NEW,OPEN" `

Reply:
We can do both

Reply:
https://stackoverflow.com/a/2602127/1461196
Convention is to use separate params though